test_sets/app_services.lua

```diff
@@ -0,0 +1,57 @@
+./test_scripts/AppServices/OnAppServiceData/002_M2H_success_flow.lua
+./test_scripts/AppServices/OnAppServiceData/004_OnAppServiceData_unknown_param_success.lua
+./test_scripts/AppServices/OnAppServiceData/003_H2M_success_flow.lua
+./test_scripts/AppServices/OnAppServiceData/006_M2M_Weather_success_flow.lua
+./test_scripts/AppServices/OnAppServiceData/001_M2M_success_flow.lua
```

Comment:
Minor detail, but these should probably be sorted by prefix. You could probably use a "Sort Lines" function with a text editor